### DISCUSSION TITLE
Auto refresh not working (unraid)

### DISCUSSION THREAD

**Discussion by brownphotographic** [advplyr/audiobookshelf#3737] [Q&A]
(moved from a reddit thread)

I have set up my podcasts in Audiobookshelf. Running in a docker container (the 'official' one in app store).

I imported from OPML, which worked.

Problems:
1. Automatic downloads of podcasts not workin.  Last Time it synced was five days ago despite there being multiple updates to the podcasts available. Forcing a manual sync doesn't work.

2. Manual addition of podcasts doesn't work. I get error message when adding.

Is this perhaps a permissions issue ? Or an issue due to the filesystem I am using (ZFS)?

Crappy screen recording attached. I find the interface VERY confusing about where to do to sync a podcast btw, hence the hunting around. Not obvious.

Log:

`text  error  warn  system  array  login  

      protocol: 'https:',
      path: '/us/podcast/the-daily/id1200361736',
      method: 'GET',
      headers: [Object],
      agent: [Agent],
      agents: [Object],
      auth: undefined,
      hostname: 'podcasts.apple.com',
      port: null,
      nativeProtocols: [Object],
      pathname: '/us/podcast/the-daily/id1200361736'
    },
    _ended: true,
    _ending: true,
    _redirectCount: 0,
    _redirects: [],
    _requestBodyLength: 0,
    _requestBodyBuffers: [],
    _eventsCount: 3,
    _onNativeResponse: [Function (anonymous)],
    _currentRequest: ClientRequest {
      _events: [Object: null prototype],
      _eventsCount: 7,
      _maxListeners: undefined,
      outputData: [],
      outputSize: 0,
      writable: true,
      destroyed: false,
      _last: true,
      chunkedEncoding: false,
      shouldKeepAlive: false,
      maxRequestsOnConnectionReached: false,
      _defaultKeepAlive: true,
      useChunkedEncodingByDefault: false,
      sendDate: false,
      _removedConnection: false,
      _removedContLen: false,
      _removedTE: false,
      strictContentLength: false,
      _contentLength: 0,
      _hasBody: true,
      _trailer: '',
      finished: true,
      _headerSent: true,
      _closed: false,
      socket: [TLSSocket],
      _header: 'GET /us/podcast/the-daily/id1200361736 HTTP/1.1\r\n' +
        'Accept: application/rss+xml, application/xhtml+xml, application/xml, */*;q=0.8\r\n' +
        'User-Agent: audiobookshelf (+https://audiobookshelf.org; like iTMS)\r\n' +
        'Host: podcasts.apple.com\r\n' +
        'Connection: close\r\n' +
        '\r\n',
      _keepAliveTimeout: 0,
      _onPendingData: [Function: nop],
      agent: [Agent],
      socketPath: undefined,
      method: 'GET',
      maxHeaderSize: undefined,
      insecureHTTPParser: undefined,
      joinDuplicateHeaders: undefined,
      path: '/us/podcast/the-daily/id1200361736',
      _ended: false,
      res: null,
      aborted: false,
      timeoutCb: null,
      upgradeOrConnect: false,
      parser: null,
      maxHeadersCount: null,
      reusedSocket: false,
      host: 'podcasts.apple.com',
      protocol: 'https:',
      _redirectable: [Circular *1],
      [Symbol(shapeMode)]: false,
      [Symbol(kCapture)]: false,
      [Symbol(kBytesWritten)]: 0,
      [Symbol(kNeedDrain)]: false,
      [Symbol(corked)]: 0,
      [Symbol(kOutHeaders)]: [Object: null prototype],
      [Symbol(errored)]: null,
      [Symbol(kHighWaterMark)]: 16384,
      [Symbol(kRejectNonStandardBodyWrites)]: false,
      [Symbol(kUniqueHeaders)]: null
    },
    _currentUrl: 'https://podcasts.apple.com/us/podcast/the-daily/id1200361736',
    _timeout: null,
    [Symbol(shapeMode)]: true,
    [Symbol(kCapture)]: false
  }
}
[2024-12-21 11:14:56.404] INFO: [RssFeedManager] Fetched all feeds
`

**Comment by nichwall** [comment]
I'm confused about what you mean by "keep podcast in sync". Are you meaning the auto download schedule to download new podcasts to the server?

Edit: I misread the beginning, yes you are specifically meaning downloading episodes using the automatic schedule.

- Has adding any podcasts through the interface worked?
- What podcasts are you trying to add?
- Are you running through a proxy to access the internet?
- Are you able to download the RSS feed from the shell in the container?
- What server version are you running?
- Do you have an example RSS feed of a podcast that is not working?

Does https://www.audiobookshelf.org/guides/podcasts help you understand the interface?

**Comment by brownphotographic** [comment]
Yes, that was the problem exactly...

I think I was the culprit here. I double checked my container settings and it was the way I had set up the tailscale option. I had set it to SERVE, but had not set up HTTPS in tailscale.

I can now:
- manually search for podcasts
- manually sync podcasts
- auto sync... i get an error for one of them though. Is there a way of looking up the ID that is provided in the log?

`024-12-21 17:00:00.451

INFO

[LibraryScanner] Starting library scan 7ff482fe-da84-4bf5-a04f-7203312359fa for Podcasts

2024-12-21 17:00:00.452

ERROR

[LibraryItem] Library item not found with id "93971282-058e-417f-9d5b-e080cd8fb7a2"

2024-12-21 17:00:00.452

ERROR

[CronManager] Library item 93971282-058e-417f-9d5b-e080cd8fb7a2 not found for episode check cron 0 * * * *

2024-12-21 17:00:02.710

INFO

[PodcastManager] runEpisodeCheck: "The Daily" | Last check: Sat Dec 21 2024 16:09:29 GMT-0500 (Eastern Standard Time) | No latest episode

2024-12-21 17:00:02.967

INFO

[LibraryItem] Library item "9b8ecbb2-7e01-4ae0-b853-06fd4c8fa20b" updated

2024-12-21 17:00:02.968

INFO

[PodcastManager] runEpisodeCheck: "Americast" | Last check: Sat Dec 21 2024 16:13:30 GMT-0500 (Eastern Standard Time) | Latest episode pubDate: Thu Dec 19 2024 15:57:00 GMT-0500 (Eastern Standard Time)

2024-12-21 17:00:03.618

INFO

[LibraryItem] Library item "1dd71467-abb5-43c9-8334-73237e69b463" updated

2024-12-21 17:00:03.619

INFO

[PodcastManager] runEpisodeCheck: "LINUX Unplugged" | Last check: Sat Dec 21 2024 16:31:10 GMT-0500 (Eastern Standard Time) | Latest episode pubDate: Sun Dec 15 2024 20:40:00 GMT-0500 (Eastern Standard Time)

2024-12-21 17:00:04.995

INFO

[LibraryItem] Library item "5e90340c-96e5-49ad-b17c-0e1fb0862986" updated

2024-12-21 17:00:05.000

INFO

[PodcastManager] runEpisodeCheck: "WHAT THE TRUCK?!?" | Last check: Tue Dec 17 2024 11:04:13 GMT-0500 (Eastern Standard Time) | Latest episode pubDate: Fri Dec 13 2024 13:34:00 GMT-0500 (Eastern Standard Time)

2024-12-21 17:00:06.634

INFO

[PodcastManager] Found 3 new episodes for podcast "WHAT THE TRUCK?!?" - starting download

2024-12-21 17:00:06.760

INFO

[LibraryItem] Library item "a95667db-f48b-4a71-bc0c-31412c5413d4" updated

2024-12-21 17:00:06.761

INFO

[PodcastManager] runEpisodeCheck: "This Week in Linux" | Last check: Tue Dec 17 2024 11:04:03 GMT-0500 (Eastern Standard Time) | Latest episode pubDate: Sat Dec 14 2024 21:00:00 GMT-0500 (Eastern Standard Time)

2024-12-21 17:00:07.918

INFO

[LibraryScanner] Library scan "7ff482fe-da84-4bf5-a04f-7203312359fa" completed in 0:07.0 | 0 Added | 19 Updated | 0 Missing

2024-12-21 17:00:07.944

INFO

[LibraryScan] Scan log saved "/metadata/logs/scans/2024-12-21_7ff482fe-da84-4bf5-a04f-7203312359fa.txt"

2024-12-21 17:00:08.782

INFO

[LibraryItem] Library item "a95667db-f48b-4a71-bc0c-31412c5413d4" updated

2024-12-21 17:00:08.784

INFO

[PodcastManager] Successfully downloaded podcast episode "A Very WHAT THE TRUCK?!? Christmas ‘24"

2024-12-21 17:00:10.124

INFO

[LibraryItem] Library item "5d4b3fdb-1b73-4c61-a985-8de461aa61bc" updated

2024-12-21 17:00:10.125

INFO

[PodcastManager] runEpisodeCheck: "Hard Fork" | Last check: Sat Dec 21 2024 16:10:29 GMT-0500 (Eastern Standard Time) | Latest episode pubDate: Fri Dec 20 2024 07:00:00 GMT-0500 (Eastern Standard Time)

2024-12-21 17:00:10.526

INFO

[LibraryItem] Library item "c5541fc6-f3d2-47f2-9c16-5c7d7354da97" updated

2024-12-21 17:00:10.526

INFO

[PodcastManager] runEpisodeCheck: "The Uncast Show" | Last check: Tue Dec 17 2024 11:03:53 GMT-0500 (Eastern Standard Time) | Latest episode pubDate: Tue Dec 10 2024 18:00:00 GMT-0500 (Eastern Standard Time)

2024-12-21 17:00:11.027

INFO

[LibraryItem] Library item "5b52377f-94d4-452f-afb9-90b4b42233a2" updated

2024-12-21 17:00:11.029

INFO

[PodcastManager] runEpisodeCheck: "The Robot Report Podcast" | Last check: Tue Dec 17 2024 11:03:48 GMT-0500 (Eastern Standard Time) | Latest episode pubDate: Thu Dec 12 2024 19:00:00 GMT-0500 (Eastern Standard Time)

2024-12-21 17:00:11.659

INFO

[LibraryItem] Library item "a95667db-f48b-4a71-bc0c-31412c5413d4" updated

2024-12-21 17:00:11.660

INFO

[PodcastManager] Successfully downloaded podcast episode "Port strike potential and trade under Trump with Flexport’s Ryan Petersen"

2024-12-21 17:00:11.711

INFO

[PodcastManager] Found 2 new episodes for podcast "The Robot Report Podcast" - starting download

2024-12-21 17:00:11.986

INFO

[LibraryItem] Library item "157626ed-f968-4ead-9967-1063927d62a8" updated

2024-12-21 17:00:11.987

INFO

[PodcastManager] runEpisodeCheck: "99% Invisible" | Last check: Sat Dec 21 2024 16:16:30 GMT-0500 (Eastern Standard Time) | Latest episode pubDate: Fri Dec 20 2024 11:29:19 GMT-0500 (Eastern Standard Time)

2024-12-21 17:00:12.618

INFO

[LibraryItem] Library item "56bbee6b-0b61-4b08-bec9-c83a32bc9969" updated

2024-12-21 17:00:12.619

INFO

[PodcastManager] runEpisodeCheck: "Linux & Open Source News" | Last check: Sat Dec 21 2024 16:17:18 GMT-0500 (Eastern Standard Time) | Latest episode pubDate: Sat Dec 21 2024 11:12:39 GMT-0500 (Eastern Standard Time)

2024-12-21 17:00:14.012

INFO

[LibraryItem] Library item "19c0d945-19ec-4eed-b01c-bfbcdd282283" updated

2024-12-21 17:00:14.013

INFO

[PodcastManager] runEpisodeCheck: "Linux Out Loud" | Last check: Tue Dec 17 2024 11:02:01 GMT-0500 (Eastern Standard Time) | Latest episode pubDate: Sun Dec 08 2024 11:30:00 GMT-0500 (Eastern Standard Time)

2024-12-21 17:00:14.403

INFO

[PodcastManager] Found 1 new episodes for podcast "Linux Out Loud" - starting download

2024-12-21 17:00:14.504

INFO

[LibraryItem] Library item "477d0fa8-84b2-40da-8fbf-e86fa7bd1b7f" updated

2024-12-21 17:00:14.505

INFO

[PodcastManager] runEpisodeCheck: "MIT Technology Review Narrated" | Last check: Tue Dec 17 2024 11:02:11 GMT-0500 (Eastern Standard Time) | No latest episode

2024-12-21 17:00:14.954

INFO

[PodcastManager] Found 1 new episodes for podcast "MIT Technology Review Narrated" - starting download

2024-12-21 17:00:15.002

INFO

[LibraryItem] Library item "862e4c87-c8c0-4508-9bde-d580b44daa60" updated

2024-12-21 17:00:15.003

INFO

[PodcastManager] runEpisodeCheck: "Acquired" | Last check: Tue Dec 17 2024 11:00:10 GMT-0500 (Eastern Standard Time) | Latest episode pubDate: Sun Dec 15 2024 20:57:15 GMT-0500 (Eastern Standard Time)

2024-12-21 17:00:15.064

INFO

[LibraryItem] Library item "a95667db-f48b-4a71-bc0c-31412c5413d4" updated

2024-12-21 17:00:15.065

INFO

[PodcastManager] Successfully downloaded podcast episode "Kal Freight’s fraud allegations; Make-A-Wish convoys; Vooma’s $13M Series A | WHAT THE TRUCK?!?"

2024-12-21 17:00:16.190

INFO

[LibraryItem] Library item "983fe07b-01e7-4050-907b-697e94164f41" updated

2024-12-21 17:00:16.191

INFO

[PodcastManager] runEpisodeCheck: "AI Today Podcast: Artificial Intelligence Insights, Experts, and Opinion" | Last check: Tue Dec 17 2024 11:00:15 GMT-0500 (Eastern Standard Time) | No latest episode

2024-12-21 17:00:17.112

INFO

[PodcastManager] Found 1 new episodes for podcast "AI Today Podcast: Artificial Intelligence Insights, Experts, and Opinion" - starting download

2024-12-21 17:00:17.189

INFO

[LibraryItem] Library item "207732cc-1f58-429a-83e8-50952522dcff" updated

2024-12-21 17:00:17.189

INFO

[PodcastManager] runEpisodeCheck: "Aircrew Interview" | Last check: Tue Dec 17 2024 11:00:20 GMT-0500 (Eastern Standard Time) | Latest episode pubDate: Sun Dec 08 2024 14:00:00 GMT-0500 (Eastern Standard Time)

2024-12-21 17:00:17.981

INFO

[LibraryItem] Library item "de68c845-8b7a-42b3-82cd-1ae2e45b74b5" updated

2024-12-21 17:00:17.983

INFO

[PodcastManager] runEpisodeCheck: "America Builds" | Last check: Tue Dec 17 2024 11:00:25 GMT-0500 (Eastern Standard Time) | Latest episode pubDate: Mon Jun 03 2024 09:00:00 GMT-0400 (Eastern Daylight Time)

2024-12-21 17:00:18.247

INFO

[PodcastManager] # of episodes (4) exceeds max episodes to keep (1)

2024-12-21 17:00:18.248

INFO

[PodcastManager] Deleting oldest episode "The Wild Robot: Where Nature Meets Technology with Peter Brown"

2024-12-21 17:00:18.355

INFO

[LibraryItem] Library item "832a579c-736e-4d7e-bcda-661e994d1ee9" updated

2024-12-21 17:00:18.356

INFO

[PodcastManager] runEpisodeCheck: "Ask Noah HD Video" | Last check: Tue Dec 17 2024 11:00:35 GMT-0500 (Eastern Standard Time) | No latest episode

2024-12-21 17:00:18.380

INFO

[LibraryItem] Library item "157626ed-f968-4ead-9967-1063927d62a8" updated

2024-12-21 17:00:18.381

INFO

[PodcastManager] Successfully downloaded podcast episode "Navigating the world of manufacturing logistics with FlexQube"

2024-12-21 17:00:19.779

INFO

[LibraryItem] Library item "17bc41af-baa4-415e-ba9e-67d1fc213944" updated

2024-12-21 17:00:19.780

INFO

[PodcastManager] runEpisodeCheck: "Old Iron Talk" | Last check: Tue Dec 17 2024 11:02:16 GMT-0500 (Eastern Standard Time) | No latest episode

2024-12-21 17:00:20.099

INFO

[LibraryItem] Library item "468660b2-3c97-47fd-99e2-c71e291cdbb9" updated

2024-12-21 17:00:20.099

INFO

[PodcastManager] runEpisodeCheck: "Open Source Creative" | Last check: Tue Dec 17 2024 11:02:21 GMT-0500 (Eastern Standard Time) | No latest episode

2024-12-21 17:00:20.711

INFO

[LibraryItem] Library item "7517020d-3423-40fa-8eb3-a36618b5fdf7" updated

2024-12-21 17:00:20.712

INFO

[PodcastManager] runEpisodeCheck: "Practical AI: Machine Learning, Data Science, LLM" | Last check: Tue Dec 17 2024 11:02:26 GMT-0500 (Eastern Standard Time) | No latest episode

2024-12-21 17:00:20.854

INFO

[PodcastManager] # of episodes (4) exceeds max episodes to keep (1)

2024-12-21 17:00:20.855

INFO

[PodcastManager] Deleting oldest episode "Funding the next wave of robotics"

2024-12-21 17:00:20.911

INFO

[LibraryItem] Library item "157626ed-f968-4ead-9967-1063927d62a8" updated

2024-12-21 17:00:20.912

INFO

[PodcastManager] Successfully downloaded podcast episode "2024 year in review: bold bets and dashed dreams"

2024-12-21 17:00:21.694

INFO

[PodcastManager] Found 1 new episodes for podcast "Practical AI: Machine Learning, Data Science, LLM" - starting download

2024-12-21 17:00:21.735

INFO

[LibraryItem] Library item "6dd70942-f704-4f84-86f9-a4bac9376f66" updated

2024-12-21 17:00:21.735

INFO

[PodcastManager] runEpisodeCheck: "The Rest Is History" | Last check: Tue Dec 17 2024 11:03:32 GMT-0500 (Eastern Standard Time) | Latest episode pubDate: Sun Dec 15 2024 19:10:00 GMT-0500 (Eastern Standard Time)

2024-12-21 17:00:22.301

INFO

[PodcastManager] Found 1 new episodes for podcast "The Rest Is History" - starting download

2024-12-21 17:00:23.143

INFO

[LibraryItem] Library item "ff228e7b-42bd-40e8-976a-6c278650ad87" updated

2024-12-21 17:00:23.148

INFO

[PodcastManager] runEpisodeCheck: "The Rest Is Politics: US" | Last check: Tue Dec 17 2024 11:03:38 GMT-0500 (Eastern Standard Time) | Latest episode pubDate: Fri Dec 13 2024 00:00:00 GMT-0500 (Eastern Standard Time)

2024-12-21 17:00:23.552

INFO

[PodcastManager] Found 1 new episodes for podcast "The Rest Is Politics: US" - starting download

2024-12-21 17:00:23.621

INFO

[LibraryItem] Library item "d3cd33c9-065a-4e66-8bc4-154578535afb" updated

2024-12-21 17:00:23.622

INFO

[PodcastManager] runEpisodeCheck: "The Road to Autonomy" | Last check: Tue Dec 17 2024 11:03:43 GMT-0500 (Eastern Standard Time) | Latest episode pubDate: Sat Dec 14 2024 10:00:00 GMT-0500 (Eastern Standard Time)

2024-12-21 17:00:23.686

INFO

[LibraryItem] Library item "477d0fa8-84b2-40da-8fbf-e86fa7bd1b7f" updated

2024-12-21 17:00:23.688

INFO

[PodcastManager] Successfully downloaded podcast episode "202: Techie Tidings | Linux Out Loud 104"

2024-12-21 17:00:25.245

INFO

[PodcastManager] Found 2 new episodes for podcast "The Road to Autonomy" - starting download

2024-12-21 17:00:25.276

INFO

[LibraryItem] Library item "862e4c87-c8c0-4508-9bde-d580b44daa60" updated

2024-12-21 17:00:25.277

INFO

[PodcastManager] Successfully downloaded podcast episode "Meet the radio-obsessed civilian shaping Ukraine’s drone defense"

2024-12-21 17:00:25.424

INFO

[LibraryItem] Library item "3c8c05ad-10e5-469a-bfe1-7015ad54586a" updated

2024-12-21 17:00:25.425

INFO

[PodcastManager] runEpisodeCheck: "Self-Hosted" | Last check: Tue Dec 17 2024 11:02:36 GMT-0500 (Eastern Standard Time) | Latest episode pubDate: Fri Dec 13 2024 05:00:00 GMT-0500 (Eastern Standard Time)

2024-12-21 17:00:26.027

INFO

[LibraryItem] Library item "4fa18340-1258-4d4a-a0d1-e02b34b36ebe" updated

2024-12-21 17:00:26.028

INFO

[PodcastManager] runEpisodeCheck: "Stuff You Should Know" | Last check: Tue Dec 17 2024 11:02:41 GMT-0500 (Eastern Standard Time) | Latest episode pubDate: Sat Dec 14 2024 05:00:00 GMT-0500 (Eastern Standard Time)

2024-12-21 17:00:27.822

INFO

[PodcastManager] Found 3 new episodes for podcast "Stuff You Should Know" - starting download

2024-12-21 17:00:27.930

INFO

[LibraryItem] Library item "53a3131a-41ee-4c76-86d0-e9abd7794334" updated

2024-12-21 17:00:27.931

INFO

[PodcastManager] runEpisodeCheck: "Talking Machines" | Last check: Tue Dec 17 2024 11:02:47 GMT-0500 (Eastern Standard Time) | No latest episode

2024-12-21 17:00:28.151

INFO

[LibraryItem] Library item "207732cc-1f58-429a-83e8-50952522dcff" updated

2024-12-21 17:00:28.152

INFO

[PodcastManager] Successfully downloaded podcast episode "AI Use Case Series: AI in Healthcare [AI Today Podcast]"

2024-12-21 17:00:28.434

INFO

[LibraryItem] Library item "13e3dbeb-461d-4347-89ca-197f2ad46216" updated

2024-12-21 17:00:28.434

INFO

[PodcastManager] runEpisodeCheck: "Tech Life" | Last check: Tue Dec 17 2024 11:02:52 GMT-0500 (Eastern Standard Time) | No latest episode

2024-12-21 17:00:28.882

INFO

[PodcastManager] Found 1 new episodes for podcast "Tech Life" - starting download

2024-12-21 17:00:28.925

INFO

[LibraryItem] Library item "0ebc47c8-d29f-45e3-b491-664b1a967221" updated

2024-12-21 17:00:28.926

INFO

[PodcastManager] runEpisodeCheck: "Techdirt" | Last check: Tue Dec 17 2024 11:02:57 GMT-0500 (Eastern Standard Time) | No latest episode

2024-12-21 17:00:29.950

INFO

[LibraryItem] Library item "aef12fe2-bc3a-49ac-a1b4-c75cdef21058" updated

2024-12-21 17:00:29.951

INFO

[PodcastManager] runEpisodeCheck: "Things That Go Boom" | Last check: Tue Dec 17 2024 11:03:58 GMT-0500 (Eastern Standard Time) | No latest episode

2024-12-21 17:00:30.676

INFO

[LibraryItem] Library item "494c25e5-2f38-4726-9bb3-c6dfccf68e57" updated

2024-12-21 17:00:30.676

INFO

[PodcastManager] runEpisodeCheck: "Global News Podcast" | Last check: Sat Dec 21 2024 16:32:06 GMT-0500 (Eastern Standard Time) | No latest episode

2024-12-21 17:00:31.031

INFO

[LibraryItem] Library item "6dd70942-f704-4f84-86f9-a4bac9376f66" updated

2024-12-21 17:00:31.032

INFO

[PodcastManager] Successfully downloaded podcast episode "Mozart to Megadeath at CHRP"

2024-12-21 17:00:31.084

INFO

[LibraryItem] Library item "73362ec8-7651-4487-8d69-8b6ea6228969" updated

2024-12-21 17:00:34.499

INFO

[LibraryItem] Library item "ff228e7b-42bd-40e8-976a-6c278650ad87" updated

2024-12-21 17:00:34.507

INFO

[PodcastManager] Successfully downloaded podcast episode "523. Charlemagne: Return of the Kings (Part 1)"

2024-12-21 17:00:40.212

INFO

[LibraryItem] Library item "d3cd33c9-065a-4e66-8bc4-154578535afb" updated

2024-12-21 17:00:40.214

INFO

[PodcastManager] Successfully downloaded podcast episode "50. President Elon Musk"

2024-12-21 17:00:42.031

INFO

[LibraryItem] Library item "3c8c05ad-10e5-469a-bfe1-7015ad54586a" updated

2024-12-21 17:00:42.033

INFO

[PodcastManager] Successfully downloaded podcast episode "Episode 256 | Autonomy Markets: Apple to Revive Robotaxi Ambitions? Waymo Expands to Japan, Uber’s Fleet Strategy"

2024-12-21 17:00:44.161

INFO

[LibraryItem] Library item "3c8c05ad-10e5-469a-bfe1-7015ad54586a" updated

2024-12-21 17:00:44.163

INFO

[PodcastManager] Successfully downloaded podcast episode "Episode 255 | May Mobility: Scaling Autonomous Vehicles in Japan with NTT and Toyota"

2024-12-21 17:00:47.866

INFO

[LibraryItem] Library item "53a3131a-41ee-4c76-86d0-e9abd7794334" updated

2024-12-21 17:00:47.867

INFO

[PodcastManager] Successfully downloaded podcast episode "Selects: Frances Perkins: Influential and Unknown"

2024-12-21 17:00:50.924

INFO

[LibraryItem] Library item "53a3131a-41ee-4c76-86d0-e9abd7794334" updated

2024-12-21 17:00:50.925

INFO

[PodcastManager] Successfully downloaded podcast episode "The Trivial Pursuit Trivia Edition "

2024-12-21 17:00:53.601

INFO

[LibraryItem] Library item "53a3131a-41ee-4c76-86d0-e9abd7794334" updated

2024-12-21 17:00:53.602

INFO

[PodcastManager] Successfully downloaded podcast episode "Short Stuff: Feed A Cold, Starve a Fever"

2024-12-21 17:00:56.904

INFO

[LibraryItem] Library item "0ebc47c8-d29f-45e3-b491-664b1a967221" updated

2024-12-21 17:00:56.905

INFO

[PodcastManager] Successfully downloaded podcast episode "Does the robotaxi have a future?"

2024-12-21 17:19:51.287

INFO

[SocketAuthority] Socket n2AjgrgHn7I9DUVEAAAG disconnected from client "rob" after 4321126ms (Reason: ping timeout)

2024-12-21 17:21:37.731

INFO

[SocketAuthority] Socket Connected to /socket.io wGRQqncb5gxWic_yAAAI

2024-12-21 17:24:24.710

INFO

[LibraryItem] Library item "73362ec8-7651-4487-8d69-8b6ea6228969" updated

2024-12-21 17:24:24.798

INFO

[PodcastManager] checkAndDownloadNewEpisodes for "Global News Podcast" - Last episode check: Thu Dec 19 2024 17:00:00 GMT-0500 (Eastern Standard Time)

2024-12-21 17:24:25.231

INFO

[PodcastManager] Found 3 new episodes for podcast "Global News Podcast" - starting download

2024-12-21 17:24:25.280

INFO

[LibraryItem] Library item "73362ec8-7651-4487-8d69-8b6ea6228969" updated

2024-12-21 17:24:27.005

INFO

[LibraryItem] Library item "73362ec8-7651-4487-8d69-8b6ea6228969" updated

2024-12-21 17:24:27.007

INFO

[PodcastManager] Successfully downloaded podcast episode "The Happy Pod: Playing the piano saved my life"

2024-12-21 17:24:28.598

INFO

[LibraryItem] Library item "73362ec8-7651-4487-8d69-8b6ea6228969" updated

2024-12-21 17:24:28.599

INFO

[PodcastManager] Successfully downloaded podcast episode "Fatalities and injuries in German Christmas market attack"

2024-12-21 17:24:30.048

INFO

[LibraryItem] Library item "73362ec8-7651-4487-8d69-8b6ea6228969" updated

2024-12-21 17:24:30.049

INFO

[PodcastManager] Successfully downloaded podcast episode "US sends diplomats to Syria"`

**Reply by nichwall** [reply]
You can look in the database to see what podcast the ID goes to, or click through all of your podcasts in the web client to see which one it is. The UUID shows up in the URL.

I would suggest looking at the database because there may be something wrong in the database that prevents the podcast from showing up in the web client or some other error.
